ember.md

```diff
@@ -359,10 +361,22 @@ export default DS.Model.extend({
   firstName: attr('string'),
   lastName: attr('string'),
 
-  // Associations
+  // Relationships
```

Comment:
Thank you for updating the terminology here to match the terminology of the linter :)

Reply:
Do we care about the order of `hasMany` vs. `belongsTo`?  Might be nice to add that to the example too?

Reply:
I'm not really concerned about the order, but I do think it's a good idea to codify an order here @andreafrost.

Reply:
I know the linter doesn't enforce an explicit order, but I still think it would be nice to use this opportunity to add a preference, i.e. `belongsTo` before `hasMany`.

@kculafic could you add a `belongsTo` line above `children: hasMany('child')` to hint at the preference?

Reply:
Sure 👍